features/step_definitions/home_steps.rb

```diff
@@ -5,7 +5,7 @@
 end
 
 Alors("le panneau {string} ne contient pas {string}") do |title, text|
-  expect(page.find(".gray-panel", text: title)).not_to have_content(text)
+  expect(page.find(".gray-panel", text: title)).to have_no_content(text)
```

Comment:
`have_no_content` ça veut pas dire "est vide" ? C'est super bizarre comme wording :thinking:

Reply:
Bon j'ai été checker, c'est legit. Juste bizarre comme phrase de dire `I expect my page to have no content "the content"` je trouve